src/karma.config.js

```diff
@@ -118,6 +123,15 @@ module.exports = function (config) {
         [
           'babelify',
           {
+            presets: [
+              '@babel/preset-env',
+              [
+                '@babel/preset-react',
+                {
+                  pragma: 'createElement',
+                },
+              ],
+            ],
```

Comment:
Its not and i meant to circle back and figure out why but forgot to do this. I'll have to do some digging around and figure out why the config is not being picked up in the subpackage, and/or why is its own config is also not being used.

Reply:
What bothered me with duplicating this config here is not the initial duplication but that it is something that could get out of sync with the config that is used elsewhere.

I can reproduce the issue you found though where compiling JSX fails if this is not specified. I think we can land this PR but note in a comment here that there is an unresolved issue that we shouldn't need to configure `presets` here. Of course, if you can figure out what the issue is that would be better.

Reply:
Another option would be to change the presets config to this:

```
presets: require('../frontend-shared/babel.config.json').presets,
```

At least then it would avoid duplicating the config.

Reply:
Good suggestion! And I filed a ticket for this issue to follow up later.